pub.go

```diff
@@ -60,6 +61,11 @@ func (pub *pubSocket) Type() SocketType {
 	return pub.sck.Type()
 }
 
+// Addr returns the listen address
```

Comment:
please adapt with whatever updated `zmq4.Socket.Addr` documentation you came up with.